Library/SubscriptionAddOn.cs

```diff
@@ -14,9 +16,10 @@ public SubscriptionAddOn(XmlTextReader reader)
             ReadXml(reader);
         }
 
-        public SubscriptionAddOn(string addOnCode, int unitAmountInCents, int quantity = 1)
+        public SubscriptionAddOn(string addOnCode, AddOn.Type addOnType, int unitAmountInCents, int quantity = 1)
```

Comment:
I'm not sure this is the way to go here because it breaks the contract. Maybe instead of changing the `SubscriptionAddOn` constructor, we should add another one. Something like this:

```csharp
// keep old constructor
public SubscriptionAddOn(string addOnCode, int unitAmountInCents, int quantity = 1)
{
  AddOnCode = addOnCode;
  UnitAmountInCents = unitAmountInCents;
  Quantity = quantity;
}

// add new constructor
public SubscriptionAddOn(string addOnCode, AddOn.Type addOnType, int unitAmountInCents, int quantity = 1)
{
  AddOnCode = addOnCode;
  AddOnType = addOnType;
  UnitAmountInCents = unitAmountInCents;
  Quantity = quantity;
}
```

What do you think?